qiita_db/metadata_template/prep_template.py

```diff
@@ -75,6 +75,27 @@ class PrepTemplate(MetadataTemplate):
     _sample_cls = PrepSample
     _fp_id = convert_to_id("prep_template", "filepath_type")
     _filepath_table = 'prep_template_filepath'
+    _log_table = "prep_template_edit"
+
+    @staticmethod
+    def _update_analyses(pt_id):
```

Comment:
Why is this a staticmethod? You're only applying it to a single prep template, so it is, by definition of the method, an instance method.